lessons/absolute-value-and-opposites/langs/en-us/index.adoc

```diff
@@ -0,0 +1,125 @@
+- Do you think that zero has an opposite?
+** _Zero is its own opposite._
+
+=== Investigate
+
+Explain to students that the function used to represent "the opposite" is called `negate`. It can be used like any other operator that we put at the top of a Circle of Evaluation - with one important difference: when we use `negate`, we can include just one argument inside the Circle.
+
+[.lesson-instruction]
+--
+- Take a look at the Circle below. What does it evaluate to?
+** _The Circle below evaluates to @math{-2}._
+
+[.image]
+@show{(coe '(negate 2))}
+@clear
+
+- What happens if we nest a negating Circle _inside of_ another negating Circle?
+** _The opposite of the opposite of a number is the number itself._
+
+[.image]
+@show{(coe '(negate (negate 2)))}
+@clear
+
+- Practice applying `negate` on Negate: True or False?
+--
+
+When we want to write a mathematical expression that indicates that negating is happen, we just use a negative sign! So, while we can read @math{-45} by simply saying "negative 45," another way to read that value's name is by saying, "the opposite of 45."
+
+=== Synthesize
+
+- Two opposites are 8 units apart from each other. What are they?
+** @math{4} and @math{-4}
+- Two opposites are 42.5 units apart from each other. What are they?
+** @math{21.5} and @math{-21.5}
+- Cia says that opposites are always negative. Are they right? Why?
+** _No, opposites are not always negative. When we negate a negative value, we get a positive value. Additionally, zero is its own opposite, and it is neither positive nor negative._
+
+== Absolute Value @duration{30 minutes}
+
+=== Overview
+
+Students consider the meaning of absolute value, and apply the concept to Circles of Evaluation using `num-abs`.
+
+=== Launch
+
+[.lesson-instruction]
+What is the distance between these two points on the number line: @math{-18} and @math{5}?
+
+Give students a minute to contemplate, and then invite them to verbally share their strategies. Record students' thinking the board, annotating the number line. All strategies are welcome, with a special interest in any discussion that hones in on the idea of _the distance of a number from zero._
+
+Explain to students that we have a term for _the distance of a number from zero_ - it's absolute value.
+
+We annotate absolute value like this: @math{|x|}, with @math{x} being any given number. When we encounter an expression like @math{|x|}, we say "the absolute value of @math{x}."
+
+Because _opposites_ are the same distance away from zero, they will always have the same absolute value. So, @math{|4| = 4} and @math{|-4| = 4}.
+
+// include number line diagram showing a 4-unit jump from 0 to 4, and from 0 to -4 //
+
+=== Investigate
+
+The function to represent absolute value is `num-abs`. It can be used like any other operator that we put at the top of a Circle of Evaluation - with one important difference: when we use `negate`, we include just one argument inside the Circle.
```

Comment:
Given that you already asked to make `negate` a function, is it worth also defining `abs`?